common/djangoapps/student/models.py

```diff
@@ -288,6 +288,8 @@ class Meta(object):
         blank=True, null=True, max_length=6, db_index=True,
         choices=LEVEL_OF_EDUCATION_CHOICES
     )
+    profession = models.TextField(blank=True, null=True)
+    specialty = models.TextField(blank=True, null=True)
```

Comment:
Thank you! I change it so profession/specialty data is stored in the UserProfile.meta field